GNUmakefile

```diff
@@ -352,8 +361,13 @@ endif
 
 .PHONY: proto
 proto:
-	@$(SHELL) $(CURDIR)/build-support/scripts/protobuf.sh \
-		--protoc-version "$(PROTOC_VERSION)"
+	@$(SHELL) $(CURDIR)/build-support/scripts/protobuf.sh 
+			
+proto-format:
```

Comment:
```suggestion
.PHONY: proto-format
proto-format: proto-tools
```

Reply:
Marking the target as PHONY disables the weird makefile "targets are output files" detection logic.

`proto-tools` will install/update `buf` as needed and is a cheap no-op if it already matches